### ISSUE TITLE
Package Not Installing

### ISSUE THREAD

**Issue by Hasnonname** [noseglid/atom-build#573]
I'm new to Atom and the Rust programming language, and I was told to use atom-build among other packages to build my code. However, when I try to install it through Install tab in Settings, the install doesn't seem to end, even when left for several hours. I've already tried doing `ipconfig /flushdns`, but that doesn't help. Please try putting your solutions in layman's terms, since I am new to Atom and don't understand much of the terminology. Thanks in advance.

**Comment by NiclasEsser1** [comment]
I have the same issue. I am able to install any package i tried, just build does not install. It ends up with 
`Installing build to C:\Users\niclas.esser\.atom\packages failed
Request for package information failed: connect ETIMEDOUT 52.22.213.157:443 (5 attempts) (ETIMEDOUT)`

**Comment by Cathabad** [comment]
I am having the same issue on Windows 10 (Build 17134).  Tried running Atom as administrator.  Did the apm build from an elevated cmd prompt, all with the same results, it hangs forever.  I can see some files are grabbed:

c:\users\name\.atom\.apm\build\0.70.0\package.tgz
c:\users\name\.atom\.apm\build\0.70.0\package\package.json 

but then nothing after that.  Atom doesn't list any dependencies for build, so not sure if my Windows 10 is just lacking some component needed or not.

I do have build 0.70.0 package installed in Atom and running fine on my MacBook Pro, and honestly I prefer coding there, but I don't always have that available and I'm trying to replicate the setup on my home PC.

Any assistance would be much appreciated.
